src/main/java/command/Command.java

```diff
@@ -0,0 +1,100 @@
+package command;
+
+import task.TaskType;
+import time.Time;
+
+public class Command {
+
+    private CommandType cmdType;
+    private Time taskTimeInfo;
+    private TaskType taskType;
+    private String taskDescription;
+
+    /**
```

Comment:
Remove redundant comments. The comment `Constructor` doesnt add any value or improve readability. You should rather provide a description (what and why) in the comments. (Consider this comment for all such redundant comments before constructor and methods)